types/async-busboy/async-busboy-tests.ts

```diff
@@ -8,20 +8,18 @@ type Result = asyncBusboy.Result;
 type OnFileResult = asyncBusboy.OnFileResult;
 type FileReadStream = asyncBusboy.FileReadStream;
 
-const req = null as any as IncomingMessage;
+const req = (null as any) as IncomingMessage;
 
 const resultPromise = asyncBusboy(req); // $ExpectType Promise<Result>
 asyncBusboy(req, {}); // $ExpectType Promise<Result>
 asyncBusboy(req, { headers: { 'content-type': 'foo' } }); // $ExpectType Promise<Result>
 asyncBusboy(req, { highWaterMark: 10 }); // $ExpectType Promise<Result>
 
 const onFileResultPromise = asyncBusboy(req, {
-    onFile: (fieldname, file, filename, encoding, mimetype) => {
-        fieldname; // $ExpectType string
-        file; // $ExpectType Readable
-        filename; // $ExpectType string
-        encoding; // $ExpectType string
-        mimetype; // $ExpectType string
+    onFile: (name, stream, info) => {
```

Comment:
is that correct?
https://github.com/m4nuC/async-busboy/blob/master/index.js#L111
https://github.com/m4nuC/async-busboy#asyncawait-using-custom-onfile-handler-ie-no-temp-files

Reply:
this is one of the questions I want to ask you. How I can use an older version of the package (busboy) in this package (async-busboy)? The package still uses older version https://github.com/m4nuC/async-busboy/blob/master/package.json#L23